src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -2,54 +2,82 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core'
 import { RundownService } from '../../../core/abstractions/rundown.service'
 import { Rundown } from '../../../core/models/rundown'
 import { DialogService } from '../../../shared/services/dialog.service'
+import { ContextMenuOption } from '../../../shared/abstractions/context-menu-option'
 
 @Component({
   selector: 'sofie-rundown-header-context-menu',
   templateUrl: './rundown-header-context-menu.component.html',
-  styleUrls: ['./rundown-header-context-menu.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RundownHeaderContextMenuComponent {
   @Input()
-  public rundown?: Rundown
+  public rundown: Rundown
+
+  @Input()
+  public shortenedRundownName: string
+
+  private readonly contextMenuOptionsForInactiveRundown: ContextMenuOption[] = [
+    {
+      label: 'Activate (On Air)',
+      contextAction: (): void => this.openActivationDialog(),
+    },
+    {
+      label: 'Reset Rundown',
+      contextAction: (): void => this.resetRundown(),
+    },
```

Comment:
This one is duplicated for the `contextMenuOptionsForActiveRundown`.   
Perhabs make a `commonContextMenuOptions`, which contains any overlap, and with spreed operator is put into the others. 

Although making that `commonContextMenuOptions`, might make it harder to order them... Adding a priority atribute could fix that, ordering the options from e.g lowest to highest.

Just a suggestion. Not that critical when there is only one that overlap, but as soon as there is 2-3 or more, then this might make sense. This might already be the case as soon as the 'Reload INews data', or what ever it is going to be named, is added.

Reply:
Regarding the context menu. I would very much like to remove these components and have a single context menu component, and a service which is responsible for setting the correct options. That will also fix the issue I showed you with multiple context menus popping up on activating a rundown. But I think it is out of scope for this PR.